### PR TITLE
  Christian/feat/add login page

### PR DESCRIPTION
**Pull request by ui-Auxilary**
Fixed up some bugs in the Landing page ie. gap at the footer, and animation not repeating for Circle logo

- Added login page

### CODE REVIEW

**Review by imagine-hussain — COMMENTED** [review]
This is looking great!

Just some minor points:

- at medium laptop width (around the 1200-1024 px mark), the horizontal margins on the container are probably a little big and make the login look smaller than we want.


image

- Since removing google, we should now be replacing that with a "continue as guest" option

**Comment by imagine-hussain** [comment]
Forgot to mention, this should also have a dark-mode implementation, though that doesn't necessarily need to be done this PR since this isn't directly going to production, that would be the next thing to look at.

File: frontend/src/pages/Auth/Container/Container.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <S.Wrapper>
+        <S.Left>
+          <S.SplashArt src={SplashArt} />
+        </S.Left>
+        <S.Right>
+          <S.Login>
+            <Link to="/">
+              <S.Back src={BackButton} />
+            </Link>
+            <h2>Login to circles</h2>
```
Comment (leonardo-fan) [inline frontend/src/pages/Auth/Container/Container.tsx]: "Login to Circles"

**Review by leonardo-fan — CHANGES_REQUESTED** [review]
Looks awesome! same comment as Hussain on the L & R margins. One grammar thing in comments. Also thoughts on keeping the login container centered vertically as well?

File: frontend/src/pages/Auth/Auth.tsx [hunk 2]
```diff
@@ -34,97 +40,65 @@ const Auth = () => {
...
-      <div onClick={logout}>
-        <h1> LOGOUT </h1>
-      </div>
-    );
-  }
   interface Props {
-    obj: any
+    obj: any;
   }
   const JsonObject = ({ obj }: Props) => {
```
Comment (ollibowers) [inline frontend/src/pages/Auth/Auth.tsx]: If no progress was made towards google login, should we just remove this?

File: frontend/src/pages/Auth/Auth.tsx [hunk 3]
```diff
@@ -34,97 +40,65 @@ const Auth = () => {
   // };
 
   const handleCallbackResponse = (response: any) => {
```
Comment (ollibowers) [inline frontend/src/pages/Auth/Auth.tsx]: Google?

File: frontend/src/pages/Auth/Auth.tsx [hunk 4]
```diff
@@ -34,97 +40,65 @@ const Auth = () => {
...
-    console.log("callBackResponse with jwt: ", jwt);
+    console.log('callBackResponse with jwt: ', jwt);
     setUserObject(jwt_decode(jwt));
 
-    
     return 0;
-  }
+  };
 
   useEffect(() => {
```
Comment (ollibowers) [inline frontend/src/pages/Auth/Auth.tsx]: Agan google

File: frontend/src/pages/Auth/Auth.tsx [hunk 5]
```diff
@@ -9,11 +9,17 @@ import type { RootState } from 'config/store';
...
 import PageTemplate from 'components/PageTemplate';
+import Container from './Container';
+import { inDev } from 'config/constants';
+import S from './styles';
+import axios from 'axios';
+import { generateUserToken, userLogin, userLogout } from 'utils/api/userApi';
 
 const Auth = () => {
   const [userObject, setUserObject] = useState({});
+  const [dummyToken, setDummyToken] = useState('');
```
Comment (ollibowers) [inline frontend/src/pages/Auth/Auth.tsx]: I don't think this state is used anymore

File: package-lock.json
Comment (ollibowers) [inline package-lock.json:1]: i dont know if this should be here (it is not the real package-lock.json)

**Review by ollibowers — CHANGES_REQUESTED** [review]
oops i did the changes as comments

File: frontend/src/pages/Auth/Auth.tsx (hunk 3 above)
Comment (imagine-hussain) [reply frontend/src/pages/Auth/Auth.tsx]: Googlie die

**Review by imagine-hussain — APPROVED** [review]
LGTM!!!

**Comment by imagine-hussain** [comment]
I wonder if I have to manually merge this because of failing pipelines of if this should work

**Comment by Peedee2002** [comment]
This has been merged

**Pull request closed without merging**